src/tasks/WasmBuildTasks/GetChromeVersions.cs

```diff
@@ -90,6 +97,31 @@ private async Task<bool> ExecuteInternalAsync()
         }
     }
 
+    private bool AreVersionsChanged(ChromeVersionSpec version, string baseUrl)
+    {
+        XmlDocument xmlDoc = new XmlDocument();
+        xmlDoc.Load(ChromeVersionsPath);
+
+        if (version.os == "Linux")
+        {
+            string linuxChromeBaseSnapshotUrl = GetNodeValue(xmlDoc, "linux_ChromeBaseSnapshotUrl");
+            string linuxV8Version = GetNodeValue(xmlDoc, "linux_V8Version");
+            return version.v8_version != linuxV8Version ||
+                baseUrl != linuxChromeBaseSnapshotUrl;
+        }
+        // version.os == "Windows"
```

Comment:
I'd prefer having an else if for the windows case and add an else+throw if os is not recognized